tests/python/Pipfile

```diff
@@ -10,6 +10,9 @@ pylint = "*"
 
 [packages]
 kin-sdk = "==2.1.3"
+aiohttp = "*"
+cchardet = "*"
```

Comment:
you don't need 'cchardet' as long as you use resp.json(encoding='utf-8') (Cause the purpose of this package if to find out the encoding even none is given)

Reply:
thanks